app/jobs/enqueue_publishers_for_payout_job.rb

```diff
@@ -2,11 +2,13 @@
 class EnqueuePublishersForPayoutJob < ApplicationJob
   queue_as :scheduler
 
-  def perform(should_send_notifications: false, final: true, payout_report_id: "", publisher_ids: [])
+  def perform(should_send_notifications: false, final: true, manual: false, payout_report_id: "", publisher_ids: [])
     Rails.logger.info("Enqueuing publishers for payment.")
 
     if publisher_ids.present?
       publishers = Publisher.where(id: publisher_ids)
+    elsif manual
+      publishers = Publisher.partner
```

Comment:
You should also scope `publisher_ids` if they're included.

Reply:
Fortunately the `if publisher_ids.present?` will receive any included id's 👍